lib/index.js

```diff
@@ -149,28 +160,37 @@ export default {
           linterName => configEnabledEngines.has(linterName));
 
         // Construct the command line invocation which runs the Code Climate CLI
-        const cmd = [this.executablePath, 'analyze',
-          '-f json',
-          makeEngineString(linterEnabledEngines),
-          `'${atom.project.relativize(filePath)}'`,
-          '< /dev/null'].join(' ');
+        const execArgs = ['analyze', '-f', 'json'].concat(
+          getEngineArgs(linterEnabledEngines),
+          [atom.project.relativize(filePath), '</dev/null']);
```

Comment:
I will need the 2nd element (last) from the array returned by `project::relativizePath`.

Let's take the filepath `/opt/cgalvarez/project/style.css` for example. `project::relativizePath` returns `["/opt/cgalvarez/project", "style.css"]`. We must feed the CodeClimate command with `"style.css"`, which is the relative path to the file to lint.

Also, the docs at the link you've provided doesn't say anything about returning `null`. While providing an absolute path as the input parameter, it will always return the array, and the `filePath` provided is the one [returned by the Atom textEditor for the currently open file](https://github.com/AtomLinter/linter-codeclimate/blob/master/lib/index.js#L90), so I think there is no need to handle that check :wink:.